script/Arcueid/Day 13/13_00_ARC13_KRAKE.txt

```diff
@@ -597,7 +597,7 @@ The enemy is wholly unknown.
 [sha:ec51f1aa2bbdee4eac3d0ec3088c1483d86e1d15]{
 -- Page 297, Offset 32490.
 -- 上級死徒に匹敵する性能である事も考慮して、
-Its abilities rival those of an upper-rank Dead Apostle.
+Its abilities rival the Greater Dead Apostles.
```

Comment:
those of the

Reply:
I think 
"Its abilities rival those of Greater Dead Apostles."
is probably the best way of putting it.
With 'the' it sounds like Greater Dead Apostles are a group onto themselves rather than just a category of powerful vampires.